### ISSUE TITLE
Implement an extension for appendFile

### ISSUE THREAD

**Issue by michaelhidalgo** [o2platform/fluentnode#91]
It would be great to have an extension to append text on top of a existing file.

**Comment by DinisCruz** [comment]
yap good idea, I just added 

```coffee
    String::file_Append = (contents, callback)->
      file_Path = @.real_Path()

      if file_Path.file_Exists() and contents
        if callback
          fs.appendFile file_Path, contents,  (err)=>
            callback err || file_Path
        else
          fs.appendFileSync file_Path, contents
      return @
```

tested by

```coffee
  it 'file_Append', (done)->
    original_Contents = tmp_File.file_Contents()
    extra_Contents_1    = "_bbbb".add_5_Letters()
    extra_Contents_2    = "_aaaa_".add_5_Letters()

    tmp_File.file_Append extra_Contents_1
    tmp_File.file_Contents().assert_Is original_Contents + extra_Contents_1
    tmp_File.file_Append extra_Contents_2, (fileName)->
      fileName.assert_Is tmp_File.real_Path()
      tmp_File.file_Contents().assert_Is original_Contents + extra_Contents_1 + extra_Contents_2
      done()
```

Is this ok

**Comment by DinisCruz** [comment]
published as v0.5.3 https://www.npmjs.com/package/fluentnode

**Comment by michaelhidalgo** [comment]
Pretty good!, thanks1
